modules/nf-core/multiqc/tests/main.nf.test.snap

```diff
@@ -2,40 +2,40 @@
     "multiqc_versions_single": {
         "content": [
             [
-                "versions.yml:md5,8f3b8c1cec5388cf2708be948c9fa42f"
+                "versions.yml:md5,b05075d2d2b4f485c0d627a5c8e475b2"
             ]
         ],
         "meta": {
-            "nf-test": "0.9.2",
+            "nf-test": "0.9.0",
             "nextflow": "24.10.4"
         },
-        "timestamp": "2025-01-27T09:29:57.631982377"
+        "timestamp": "2025-03-26T16:05:18.927925"
     },
     "multiqc_stub": {
         "content": [
             [
                 "multiqc_report.html",
                 "multiqc_data",
                 "multiqc_plots",
-                "versions.yml:md5,8f3b8c1cec5388cf2708be948c9fa42f"
+                "versions.yml:md5,b05075d2d2b4f485c0d627a5c8e475b2"
             ]
         ],
         "meta": {
-            "nf-test": "0.9.2",
+            "nf-test": "0.9.0",
             "nextflow": "24.10.4"
         },
-        "timestamp": "2025-01-27T09:30:34.743726958"
+        "timestamp": "2025-03-26T16:05:55.639955"
     },
     "multiqc_versions_config": {
         "content": [
             [
-                "versions.yml:md5,8f3b8c1cec5388cf2708be948c9fa42f"
+                "versions.yml:md5,b05075d2d2b4f485c0d627a5c8e475b2"
             ]
         ],
         "meta": {
-            "nf-test": "0.9.2",
+            "nf-test": "0.9.0",
```

Comment:
is there a reason why we downgrade nf-test?

Reply:
I think it's because I have not updated nf-test to the latest version on my computer :see_no_evil: I don't think this should be a big issue though, the nextflow version is not the latest either and this was not caught during review when I updated this in the modules repository :thinking:

https://github.com/nf-core/modules/pull/8120

Reply:
It just tells you what version was used when you ran the test, it isn't being checked in anyway.

Reply:
Perfect, thank you for the clarification :pray: